src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -16,8 +16,19 @@
 var ReactFiberReconciler = require('ReactFiberReconciler');
 var ReactGenericBatching = require('ReactGenericBatching');
 var emptyObject = require('emptyObject');
+var ReactTypeOfWork = require('ReactTypeOfWork');
```

Comment:
Please verify this file actually ends up in `build/packages/react-test-renderer` after you've run `npm run build`. Whether it does or not depends on where `ReactTypeOfWork` is located. Explanation for our folder structure: https://facebook.github.io/react/contributing/codebase-overview.html#shared-code.